src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -194,17 +244,21 @@
      * @description: starts the BDI dry run to verify DataImport__c matches
      * @param recordIds: list of DataImport__c RecordIds to check for dry run
      */
-    runDryRun: function(component, recordIds) {
+    runNewRecordDryRun: function(component, recordId) {
```

Comment:
Is there a lot of overlap between this function and handleRowEdit()? Could they share a function and pass in the action (which is the main difference I think?) as a parameter?

Reply:
What caught my eye is that the callbacks between runNewRecordDryRun and handleRowEdit are nearly identical except for the toast behavior so maybe there was a way to split that out into updateTableFromModel() or whatever name makes sense - but it's like, what six lines of code? I'm honestly not sure if the juice is worth the squeeze here.

Reply:
@ptewson-sfdo I reworked this a bit to combine them, including moving the use of 'afterDryRun' to be the callback function used and renaming the old 'afterDryRun' to be more specific to what we want to happen with the datatable rows. What do you think?

Reply:
I love it! Nice!